src/.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Actualizeaza branch-ul din **database-develop**. Trebuie sa te asiguri ca ai ultimele modificari cu gitignore.

Reply:
Au fost adaugate si alte commit-uri din cauza unui push dat din greseala de florin.Pr-ul initial nu continea elemente care nu ar trebui sa ajunga in database-develop. Din acest motiv o sa inchid acest pull request. Un nou pull request a fost creat #16 pentru aceasta implementare.